x-pack/plugins/ml/public/timeseriesexplorer/components/annotation_description_list/index.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * React component for listing pairs of information about the detector for which
+ * rules are being edited.
+ */
+
+import React from 'react';
+
+import { EuiDescriptionList } from '@elastic/eui';
+
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
+import { Annotation } from '../../../../common/types/annotations';
+
+interface Props {
+  annotation: Annotation;
+}
+
+function formatListDate(ts: number) {
+  return formatDate(ts, 'MMMM Do YYYY, HH:mm:ss');
```

Comment:
This format is being defined in several other places in our codebase - could it be moved out into a constants file now?
```
./public/components/anomalies_table/anomalies_table.js
./public/components/anomalies_table/anomaly_details.js
./public/timeseriesexplorer/forecasting_modal/forecasts_list.js
```

Reply:
Good one! I'm also using that time format in the calendars/ - feels like a good sign to move time/date formats into a shared ml ui constants file. 👍

Reply:
Ok, sure! Since it affects code outside the scope of annotations I took a note in https://github.com/elastic/kibana/issues/25856 and will do that in a follow up PR.